src/main/java/org/alliancegenome/curation_api/view/View.java

```diff
@@ -150,4 +150,7 @@ public static Class<?> viewLookup(String name) {
 		}
 		return ForPublic.class;
 	}
+
+	public class ModelDocumentView {
```

Comment:
And this view would match the class name as in GeneModelDocument.

Reply:
AffectedGenomicModelView would make sense but that already exists.

Reply:
Adding the word view onto the end of the class name is redundant.